fe1-web/src/features/evoting/network/messages/__tests__/CastVote.test.ts

```diff
@@ -147,4 +155,100 @@ describe('CastVote', () => {
       expect(msg.action).toEqual(ActionType.CAST_VOTE);
     });
   });
+
+  describe('validateVotes', () => {
+    it('returns true if all fields are defined', () => {
+      expect(() => CastVote.validateVotes([])).not.toThrow();
+      expect(() =>
+        CastVote.validateVotes([{ id: 'someId', question: 'q', vote: [0] }]),
+      ).not.toThrow();
+    });
+
+    it('returns false if some fields are undefined', () => {
+      expect(() =>
+        CastVote.validateVotes([{ id: 'someId', question: 'q', vote: undefined as any }]),
+      ).toThrow(ProtocolError);
+
+      expect(() =>
+        CastVote.validateVotes([{ id: 'someId', question: undefined as any, vote: [0] }]),
+      ).toThrow(ProtocolError);
+
+      expect(() =>
+        CastVote.validateVotes([{ id: undefined as any, question: 'q', vote: [0] }]),
+      ).toThrow(ProtocolError);
```

Comment:
Ah yes, that works too, it is just slightly more verbose since I have to cast via `unkown`. But I think you're right that this might be better since in the future we might want to add a linter rule forbidding any 🤔